source/docs/software/hardware-apis/motors/wpi-drive-classes.rst

```diff
@@ -321,6 +321,14 @@ MecanumDrive is a method provided for the control of holonomic drivetrains with
       :linenos:
       :lineno-start: 31
 
+  .. tab:: Python
+
+    .. remoteliteralinclude:: https://raw.githubusercontent.com/robotpy/examples/main/mecanum-drive/robot.py
```

Comment:
Regardless of Inspector, this should use a pinned version rather than `main`.

Reply:
We don't have versions for our examples? But I suppose we can do that. What version do you want?

Reply:
Like the allwpilib examples, it should be a git tag or commit hash.

Reply:
The issue is that examples may be updated and if lines are added or removed the frc-docs RLI will point to junk.